network-api/networkapi/buyersguide/templates/buyersguide_home.html

```diff
@@ -87,6 +87,12 @@ <h1 class="h1-heading text-center mt-4">Shop Safe This Holiday Season</h1>
         </a>
       </figure>
       {% endfor %}
+      <div class="no-matching-products-note bg-gray w-100 pt-5 d-none">
+        <div class="d-flex flex-column align-items-center">
+          <div>No results. </div>
```

Comment:
Some odd unicodeness going on here:
![image](https://user-images.githubusercontent.com/177243/49117524-d5ecad80-f255-11e8-9934-4bb01e0701e1.png)

Reply:
ummm this is what I'm seeing

<img width="706" alt="image" src="https://user-images.githubusercontent.com/2896608/49118820-b4da8b80-f25a-11e8-8dbd-6cc4adaa7404.png">

Reply:
exactly. Your system's hiding a zero-width unicode character that's secretly slipped in there =)

If I hit edit on github, I see this:

![image](https://user-images.githubusercontent.com/177243/49118956-4649fd80-f25b-11e8-99e7-5da8aed46e09.png)